ci/kind/kind-setup.sh

```diff
@@ -78,6 +84,22 @@ function get_encap_mode {
   echo "--encap-mode $ENCAP_MODE"
 }
 
+function add_option {
+     options+=(""$1" "$2"")
```

Comment:
```suggestion
     options+=("$1 $2")
```

although maybe for clarity this would be better:
```bash
function add_option {
     local option="$1"
     local action="$2"
     options+=("$option $action")
}
```